baseapp-auth/baseapp_auth/extensions/allauth/account/signup.py

```diff
@@ -0,0 +1,13 @@
+# from allauth.account.forms import SignupForm
+from django import forms
+
+
+class ExtraSignupFormFields(forms.Form):
```

Comment:
Can we change this file location to `baseapp-auth/baseapp_auth/extensions/allauth/account/forms.py` and change the class name to `SignupForm`? To match allAuth's file structure.

Ex: `from allauth.account.forms import SignupForm`